### PR TITLE
qemu.tests.multi_disk_random_hotplug: fix syntax

### PR DESCRIPTION
**Pull request by xutian**
ID: 1389155

Signed-off-by: Xu Tian xutian@redhat.com

### CODE REVIEW

File: qemu/tests/multi_disk_random_hotplug.py [hunk 1]
```diff
@@ -135,6 +135,7 @@ def insert_into_qdev(qdev, param_matrix, no_disks, params, new_devices):
         """
         dev_idx = 0
         _new_devs_fmt = ""
+        pci_bus = {'aobject': 'pci.0'}
```
Comment (suqinhuang) [inline qemu/tests/multi_disk_random_hotplug.py]: it's a workaround to fix the pci_bus issue?

Comment (xutian) [inline qemu/tests/multi_disk_random_hotplug.py]: just a fix for leak pci_bus args syntax issue

**Comment by xutian** [comment]
@suqinhuang and @CongLi any disagreements? if no I will merge this small fix.

**Comment by suqinhuang** [comment]
ok for me if it just a workaround, need to update it again when support pcie bus

**Comment by xutian** [comment]
easy syntax fix, so merged it

**Pull request merged**